bbws/entity.py

```diff
@@ -33,19 +32,55 @@
 
 
 class EntityResource(Resource):
+    get_parser = reqparse.RequestParser()
+    get_parser.add_argument('revision', type=int, default=None)
+
+    entity_class = Entity
```

Comment:
What's entity_class used for? Is it left over from when all the specific entity types were in this resource?